R/RLearner_FDAclassif_knn.R

```diff
@@ -0,0 +1,39 @@
+#' @title Learner for knn on functional data.
+#'
+#' @description
+#' Learner for knn on functional data.
+#'
+#' @export
+makeRLearner.fdaclassif.knn = function() {
+  makeRLearnerClassif(
+    cl = "fdaclassif.knn",
+    package = "fda.usc",
+    par.set = makeParamSet(
+      makeIntegerVectorLearnerParam(id = "knn", lower = 1L, default = 1L),  #FIXEME: default in fda.usc is NULL
+      makeDiscreteLearnerParam(id = "metric", default = "metric.lp", values = list("metric.lp", "metric.kl", "metric.hausdorff", "metric.dist")),
+      makeDiscreteLearnerParam(id = "type.CV", default = "GCV.S", values = list("GCV.S", "CV.S", "GCCV.S")),
+      makeUntypedLearnerParam(id = "par.CV"),  # FIXME: we don't understand this parameter
```

Comment:
```type.CV``` 
- GCV.S = Generalized Crossvalidation Score
- GCCV.S = Generalized Correlated Crossvalidation Score

```type.CV``` 
- trim : The alpha of the trimming. [0; ?]
- draw =TRUE, draw the curves, the sample median and trimmed mean. TRUE, FALSE

Suggestion: 
make own Parameters for trim and draw.

```
makeNumericLearnerParam(id = "trim"),
makeLogicalLearnerParam(id = "draw", default = TRUE),
```